package.json

```diff
@@ -13,6 +13,6 @@
     "browser-sync": "^2.18.13",
     "git": "^0.1.5",
     "jquery": "^3.2.1",
-    "sweetalert": "^2.1.0"
+    "sweetalert2": "^7.9.0"
```

Comment:
I don't think we should change to sweetalert 2 -- it looks ugly. I thought that's what we'd decided on???